bucoffea/vbfhinv/vbfhinvProcessor.py

```diff
@@ -411,12 +493,21 @@ def process(self, df):
 
             weights = candidate_weights(weights, df, evaluator, muons, electrons, photons, cfg)
 
+            # EWK corrections to VBF signal
+            if cfg.RUN.APPLY_EWK_CORR_TO_SIGNAL:
+                if re.match('VBF_HToInv.*', df['dataset']):
+                    def ewk_correction(a, b):
+                        return 1 + a * df['GenMET_pt'] + b
```

Comment:
For future use: Let's use the higgs boson from the gen collection here.
```higgs_pt = gen[(gen.pdg==25)&(gen.status==62)].max()```

Reply:
Thanks, done in 517c5e45d3c18b78f1176a8a20b4995ea9e66567